docs/documentation.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Documentation
+=============
+
+Generating Documentation
+------------------------
+This documentation is generated and published at
+`Read the Docs`_ whenever the master branch is updated.
+
+To generate locally, install the packages in ``requirements/docs.txt``, and run::
```

Comment:
More direction here would be nice. Can we install these as part of the vagrant dev environment? The jinja version numbers seem to clash.